public_html/wp-content/mu-plugins/blocks/source/blocks/schedule/front-end.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Internal dependencies
+ */
+import { getDerivedSessions } from './data';
+import { ScheduleGridContext } from './edit';
+import { ScheduleGrid } from './schedule-grid';
+import renderFrontend from '../../utils/render-frontend';
+import './front-end.scss';
+
+// See `pass_global_data_to_front_end()` for details on front- vs back-end data sourcing.
+const rawScheduleData = window.WordCampBlocks.schedule || {};
+
+/*
+ * todo
+ *
+ * this file is being loading in the editor, but should only load on the front end.
+ * probably a similar problem mentioned in init(), so maybe conditionally register it if on the front end?
+ * `wp_using_themes() && ! wcorg_is_rest_api_request()` ? - https://wordpress.stackexchange.com/a/360401/3898
+ *
+ * the front-end build file is really big
+ * still seeing a bunch of dependencies being added to it, like classnames, lodash, memoize, @babel and @emotion,
+ * etc.
```

Comment:
Over half of it is from moment-timezone. I see classnames, which makes sense since it's not a WP-provided package. lodash is correctly pulled out into the assets manifest, and I don't see these others.

Reply:
> Over half of it is from moment-timezone.

That's what i'm seeing too. Do you see a better way to work around that Gutenberg bug?

> classnames [...] makes sense since it's not a WP-provided package.

It's bundled with Core, though. If every plugin bundles their own copy of it, with different versions, wouldn't there be conflicts and unnecessary performance hits?

Reply:
> Do you see a better way to work around that Gutenberg bug?

Nope, aside from fixing the bug :) This is fine for now.

> classnames

It's 1KB, and shouldn't have any issues with conflicts. I imagine having to do an extra HTTP request would be more of a perf-hit than it's worth.

Reply:
>  shouldn't have any issues with conflicts

Why's that (now, and in the future)?

Reply:
> having to do an extra HTTP request would be more of a perf-hit than it's worth

Wouldn't it be `cat`'d with other scripts by `load-scripts.php`? I guess it also depends on HTTP1 vs HTTP2 also.

But I guess it doesn't really matter, since it doesn't look like Core exposes a discrete `classnames` script, the way it does with `moment` and `lodash`.

Reply:
> Why's that (now, and in the future)?

Each script that bundles it bundles its own local version, and it doesn't populate any globals.

> But I guess it doesn't really matter, since it doesn't look like Core exposes a discrete classnames script, the way it does with moment and lodash.

Yeah, that's what I said originally. I thought you were asking why core doesn't expose it.

Reply:
> doesn't populate any globals.

Ah, you're right, that's the difference.